Model/Api/ShippingMethods.php

```diff
@@ -298,7 +298,7 @@ private function validateEmail($email)
     public function getShippingMethods($cart, $shipping_address)
     {
         try {
-            // $this->logHelper->addInfoLog($this->request->getContent());
+//            $this->logHelper->addInfoLog($this->request->getContent());
```

Comment:
remove

Reply:
@daisy1754  this is in place for debugging purpose. maybe it could stay.

Reply:
ok fine